10-JavaScript/classes.md

```diff
@@ -44,18 +44,29 @@ When you create a new book instance the `initialize` method runs and sets the in
 Below is this Book class written in JavaScript.  It has a title, and author attributes and a `toString` method which outputs the object as a String.  Notice that everything in a class falls between the curly braces without any commas to separate each item.  
 
 ```javascript
+/**
+  *  Book class
+  */
 class Book {
```

Comment:
I don't know if we need to go quite so far as to provide docstrings.
- I think it'll be really easy to forget to do, and then our curriculum will be inconsistent
- For a small demonstration class like this, the weird syntax might actually distract from what we're trying to teach

Reply:
Yeah, I'd left one example in here intending to explain a bit about docstrings, but it's does distract from the main points.

Removed.